<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoRaSerialPortControl</class>
 <widget class="QMainWindow" name="LoRaSerialPortControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="3" column="6" alignment="Qt::AlignTop">
       <widget class="QPushButton" name="predictionStartButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Start Predictive Trackig</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5" alignment="Qt::AlignBottom">
       <widget class="QPushButton" name="stopButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>75</height>
         </size>
        </property>
        <property name="text">
         <string>Stop Tracking</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6" alignment="Qt::AlignBottom">
       <widget class="QPushButton" name="EStopButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>75</height>
         </size>
        </property>
        <property name="text">
         <string>E-Stop</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="7">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="5">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="5" alignment="Qt::AlignTop">
       <widget class="QPushButton" name="startButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Start Basic Tracking</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="3" column="0">
         <widget class="QComboBox" name="ArduinoComboBox">
          <property name="insertPolicy">
           <enum>QComboBox::InsertAlphabetically</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="LoRaSelectButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 170, 127);
</string>
          </property>
          <property name="text">
           <string>Select Serial Port</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="LoRaComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::InsertAlphabetically</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Arduino Serial Port</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="LoRaLabel">
          <property name="text">
           <string>LoRa Serial Port</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="LoRaRefreshButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 170, 127);</string>
          </property>
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="ArduinoSelectButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 170, 127);</string>
          </property>
          <property name="text">
           <string>Select Serial Port</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="ArduinoRefreshButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 170, 127);</string>
          </property>
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="7">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QPushButton" name="UpButton">
          <property name="minimumSize">
           <size>
            <width>133</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 170, 127);</string>
          </property>
          <property name="text">
           <string>Tilt Up</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="ClockWiseButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>133</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 170, 127);</string>
          </property>
          <property name="text">
           <string>Rotate Clockwise</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QTextBrowser" name="AdjustmentLogBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>508</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="DownButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 170, 127);</string>
          </property>
          <property name="text">
           <string>Tilt Down</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="CounterClockeWiseButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 170, 127);</string>
          </property>
          <property name="text">
           <string>Rotate Counter-Clockwise</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item alignment="Qt::AlignRight">
           <widget class="QComboBox" name="degreesPerClickBox">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0.5</string>
             </property>
            </item>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Degrees Per Click</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="6" column="0" colspan="7">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="SerialMonitorTab">
         <attribute name="title">
          <string>Serial Monitor</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QTextBrowser" name="statusBox">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.1pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="ClearSerialButton">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(170, 170, 127);</string>
            </property>
            <property name="text">
             <string>Clear Serial</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="MapTab">
         <attribute name="title">
          <string>Map</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="CalibrationTab">
         <attribute name="title">
          <string>Set Up</string>
         </attribute>
         <widget class="QGroupBox" name="groupBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>691</width>
            <height>101</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string>Ground Station Location</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="gridLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>691</width>
             <height>72</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Latitude:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="setGSLocationButton">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(85, 170, 255);
background-color: rgb(170, 170, 127);</string>
              </property>
              <property name="text">
               <string>Set GS Location</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="GSLongBox">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Input Longitutde...</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Longitude:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="GSLatBox">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Input Latitude...</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Altitude:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="GSAltBox">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Input Altitude...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>691</width>
            <height>191</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Allignment</string>
          </property>
          <widget class="QWidget" name="gridLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>39</y>
             <width>141</width>
             <height>141</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="2" column="0">
             <widget class="QPushButton" name="returnToSunButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(170, 170, 127);</string>
              </property>
              <property name="text">
               <string>Return to Sun Position</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="calculateStartingPosButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(170, 170, 127);</string>
              </property>
              <property name="text">
               <string>Calculate Sarting Position</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="setStartingPosButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(170, 170, 127);</string>
              </property>
              <property name="text">
               <string>Set Starting Position</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="verticalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>20</y>
             <width>261</width>
             <height>172</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Starting Azimuth</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="startingAzimuthBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>29</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>29</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Starting Elevation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="startingElevationBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>29</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>29</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="textBrowser">
              <property name="styleSheet">
               <string notr="true">background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.1pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;*First, point antenna towards Sun, then click &amp;quot;Set Starting Position&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>450</x>
             <y>20</y>
             <width>231</width>
             <height>260</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Elevation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="elevationDisplay"/>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Azimuth</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="azimuthDisplay"/>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Line of Sight Distance</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="distanceDisplay"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Commands</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
